mava/components/jax/building/rate_limiters.py

```diff
@@ -37,12 +37,26 @@ def __init__(self, config: RateLimiterConfig = RateLimiterConfig()) -> None:
         self.config = config
 
     @abc.abstractmethod
-    def on_building_rate_limiter(self, builder: SystemBuilder) -> None:
+    def on_building_data_server_rate_limiter(self, builder: SystemBuilder) -> None:
```

Comment:
Did you make sure to also update the callbacks.py and mixin class with this hook change? I don't see any changes to those files?

Reply:
So it is already called `on_building_data_server_rate_limiter` in the callbacks (https://github.com/instadeepai/Mava/blob/8af6b01f67969d5dd4889e00391aa76d943e5b60/mava/callbacks/base.py#L59). I think it was a mistakenly called `on_building_rate_limiter` here.